tests/python/contrib/test_hexagon/infrastructure.py

```diff
@@ -18,36 +18,43 @@
 """ Hexagon testing infrastructure """
 
 import tvm
+from tvm import te
 import numpy
 
 
 def ceildiv(o, d):
     return tvm.tir.floordiv(o + d - 1, d)
```

Comment:
Should we add a comment that this function might only work as intended for _positive_ numbers?

tl;dr:
It's not obvious if this has intuitive behavior when `o` and/or `d` are negative. AFAIK `o` and `d` will only be positive when we're calling this, but someone might decide to promote this function to a broader role in TVM.

Reply:
I added some asserts to catch cases where `o` or `d` are less than zero.